libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -59,6 +61,10 @@ public static NewSlackMessage ActivityToSlack(Activity activity)
                     {
                         message.Blocks = att.Content;
                     }
+                    else if (att.ContentType == HeroCard.ContentType)
+                    {
+                        message.Blocks = HeroCardToBlockKit((HeroCard)att.Content);
+                    }
```

Comment:
Note: I have a slight concern, here, that this will convert ALL hero cards to Block Kit, without giving the developer an option to disable/revert. Personally, I think this is better than the image conversion we use currently, but am very open to adding a `ConvertHeroCardsToBlockKit` flag in `SlackAdapterOptions`

Reply:
I also think that this option is better. I don't think that having an option to use the old version is going to help anyone since it was not right, at least not for what I saw. Is there a scenario where it was better than an actual Hero Card which is actually the original content?

Reply:
> Is there a scenario where it was better than an actual Hero Card which is actually the original content?

Not that I'm aware of. I just hate feeling like I'm yanking the rug out from under people, even when the adapter is more or less in a Preview state.